content/news/ragl-season1-conclusion.md

```diff
@@ -0,0 +1,52 @@
+<div style="text-align:center" markdown="1">
+![Global chat lobby](/images/news/20160329-ragl-announcement.png)
+</div>
+
+After more than three months, the very successful Season 1 of the Red Alert Global League, hosted by our community member SoScared, is officially over! Here is what he has to say in his own words:
+
+<div style="border-radius: 4px; background-color: #272d2c; padding:10px; margin:10px;">
+<p>With 184 registered matches played over the course of three months, season 1 of the Red Alert Global League is officially concluded! A big thank you for everyone participating this season, laying the cornerstone for a continuous competetive event and a venue for people to meet and play some good games, both for fun and for prestige. But before we start planning future filled stadiums on par with LoL events, let's have a closer look at how season 1 went for our participating players.</p>
```

Comment:
Could we reduce the top and bottom margins so we have a similar sized margin all round?